contribs/gmf/src/directives/partials/mobilemeasurelength.html

```diff
@@ -0,0 +1,30 @@
+<a class="btn btn-default"
+   ng-if="ctrl.drawing && (!ctrl.valid)"
```

Comment:
sorry it was for `active` not valid
but the example http://adube.github.io/ngeo/mobile-measure/examples/contribs/gmf/mobilemeasurelength.html never shows the buttons.